package.json

```diff
@@ -57,30 +57,32 @@
 		"@cosmjs/tendermint-rpc": "~0.30.0",
 		"@cosmjs/utils": "~0.30.0",
 		"@stablelib/ed25519": "^1.0.3",
+		"@types/secp256k1": "^4.0.3",
 		"cosmjs-types": "^0.7.1",
-		"did-jwt": "^6.11.6",
+		"did-jwt": "^8.0.4",
 		"did-resolver": "^4.1.0",
 		"file-type": "^16.5.4",
 		"long": "^4.0.0",
 		"multiformats": "^9.9.0",
-		"uuid": "^9.0.0"
+		"secp256k1": "^5.0.0",
+		"uuid": "^10.0.0"
 	},
 	"devDependencies": {
 		"@semantic-release/changelog": "^6.0.3",
-		"@semantic-release/commit-analyzer": "^10.0.1",
+		"@semantic-release/commit-analyzer": "^13.0.0",
 		"@semantic-release/git": "^10.0.1",
-		"@semantic-release/github": "^9.0.3",
-		"@semantic-release/npm": "^10.0.3",
-		"@semantic-release/release-notes-generator": "^11.0.2",
-		"@types/jest": "^29.5.2",
-		"@types/node": "^20.2.5",
-		"@types/uuid": "^9.0.1",
-		"conventional-changelog-conventionalcommits": "^5.0.0",
-		"jest": "^29.5.0",
-		"prettier": "^3.2.5",
-		"ts-jest": "^29.1.0",
-		"ts-node": "^10.9.1",
-		"typescript": "^5.1.3",
+		"@semantic-release/github": "^10.1.6",
+		"@semantic-release/npm": "^12.0.1",
+		"@semantic-release/release-notes-generator": "^14.0.1",
+		"@types/jest": "^29.5.12",
+		"@types/node": "^20.16.1",
+		"@types/uuid": "^10.0.0",
+		"conventional-changelog-conventionalcommits": "^8.0.0",
+		"jest": "^29.7.0",
+		"prettier": "^3.3.3",
+		"ts-jest": "^29.1.2",
+		"ts-node": "^10.9.2",
+		"typescript": "^5.5.4",
```

Comment:
Hmm I think it's missing the plain vanilla semantic-release package actually. The version would be 24.1.0 I believe on latest.

The above versions here for the dev-deps for semantic release and conventional commit are for Node v20.x. We should also look at bumping node version in package.json here and see if that works

Reply:
Credo supports `node>=18`